### DISCUSSION TITLE
Nhost Functions migrated to Node.js 18

### DISCUSSION THREAD

**Discussion by dbarrosop** [nhost/nhost#2288] [Changelog]
Dear Nhost Users,
We are pleased to inform you that the migration to Node.js 18, as mentioned in our [previous announcement](https://github.com/nhost/nhost/discussions/2239), has been successfully completed today, October 1st. This upgrade ensures improved performance, security, and compatibility for your serverless functions.

If you are deploying with a configuration file (`nhost.toml`) and you haven't done so already, please update the file to indicate that your functions are now using Node.js 18. Update the following lines:

```toml
[functions.node]
version = 18
```

Failing to do so will cause deployments to fail.

Additionally, we highly recommend updating to the latest version of our CLI, preferably version `1.8.0` or higher. Keeping your CLI up to date will ensure you have access to the latest features, bug fixes, and improvements.

We appreciate your cooperation in reviewing your code and dependencies for compatibility with Node.js 18. If you encounter any challenges during this transition or have any questions, please don't hesitate to reach out to us. We are here to assist you.

Thank you for your continued support and understanding.

Best regards,
The Nhost Team

**Comment by JIGAR-SUTARIYA** [comment]
@dbarrosop - When will Nhost functions support Node.js 20?
